### PR TITLE
adding NavBar in the header

### PR DESCRIPTION
**Pull request by AntoineCAU**
Creating NavBar links in the Header component

### CODE REVIEW

**Review by Erriep — CHANGES_REQUESTED** [review]
Change Contact to contact on the link route.

**Review by Erriep — DISMISSED** [review]

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -1,4 +1,28 @@
+import { BrowserRouter as Router, NavLink } from 'react-router-dom';
+
 function Header() {
-  return <>Ceci est mon Header</>;
+  return (
+    <Router>
```
Comment (LoicBrassart) [inline src/components/Header/index.jsx]: Your `<Router>` should be in the root `index.js` file
(Right now, **only** your Header benefits from the Router, not the whole app)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Plz fix Evil!

**Review by Erriep — DISMISSED** [review]

**Review by Erriep — DISMISSED** [review]

**Review by Erriep — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**